playbook/app/pb_kits/playbook/pb_title/_title.scss

```diff
@@ -56,4 +52,7 @@
         }
     }
   }
+  &.dark {
+    @include pb_title_dark;
+  }
```

Comment:
fix for this title responsive prop bug i found: [video](https://share.zight.com/Blu2jvqA)